packages/data-stores/src/user/reducer.ts

```diff
@@ -34,12 +34,17 @@ const newUserData: Reducer< NewUser | undefined, Action > = ( state, action ) =>
 	return state;
 };
 
-const newUserError: Reducer< NewUserErrorResponse | undefined, Action > = ( state, action ) => {
+export const newUserError: Reducer< NewUserErrorResponse | undefined, Action > = (
+	state,
+	action
+) => {
 	switch ( action.type ) {
 		case 'FETCH_NEW_USER':
 			return undefined;
 		case 'RECEIVE_NEW_USER':
 			return undefined;
+		case 'CLEAR_ERRORS':
+			return undefined;
```

Comment:
You could collapse all these cases together, but that's really just a stylistic thing.